samples/azure-maps/README.md

```diff
@@ -0,0 +1,37 @@
+# Samples: Api Management proxing calls to Azure Maps
+
+This is a sample demonstrating how to use Azure API Management (APIM) to proxy calls to the Azure Maps service. This setup allows you to manage, secure, and monitor access to Azure Maps through APIM.
```

Comment:
Can the integration between APIM and Azure Maps be simplified to use APIM's (system-assigned) MI or does this require a SAS token? I'm thinking of a similar setup to the blob storage access in terms of auth between two Azure services. The reason we return a SAS token there is because the user needs to have direct access, but here, we do not return the SAS token to the caller; we just proxy the request, I think. It may be that this turns into a similar setup as the secure blob access in that a SAS key would be returned to the caller to then call Maps directly. That's the valet key pattern.